heareval/predictions/task_predictions.py

```diff
@@ -75,7 +75,7 @@ def __len__(self):
         # return len(self.labels)
 
     def __getitem__(self, idx):
-        return self.embedding_memmap[idx]
+        return np.copy(self.embedding_memmap[idx])
```

Comment:
#142 -- added this to try. Can keep it if we want. Probably needs to be profiled.